docs/tasks.md

```diff
@@ -439,4 +439,256 @@ The following tables gives you an overview of the tasks in MTEB.
+| ron | 0 | 5 | 0 | 0 | 1 | 0 | 0 | 2 | 1 | 0 |
+| run | 0 | 2 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| rus | 0 | 6 | 2 | 0 | 0 | 3 | 0 | 5 | 1 | 0 |
+| sag | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| sah | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| san | 0 | 3 | 0 | 0 | 0 | 1 | 0 | 0 | 0 | 0 |
+| sat | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| scn | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| shn | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| sin | 0 | 3 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| slk | 0 | 2 | 0 | 0 | 1 | 0 | 0 | 2 | 0 | 0 |
+| slv | 0 | 4 | 0 | 0 | 1 | 0 | 0 | 1 | 0 | 0 |
+| smo | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| sna | 0 | 2 | 2 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| snd | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| som | 0 | 2 | 2 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| sot | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| spa | 0 | 12 | 4 | 0 | 1 | 2 | 1 | 9 | 2 | 0 |
+| sqi | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| srd | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| srp | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ssw | 0 | 3 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| sun | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| swa | 0 | 5 | 2 | 0 | 0 | 1 | 0 | 0 | 0 | 0 |
+| swe | 0 | 8 | 2 | 0 | 1 | 1 | 0 | 3 | 0 | 0 |
+| swh | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| szl | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tam | 0 | 6 | 1 | 0 | 0 | 1 | 0 | 3 | 1 | 0 |
+| taq | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tat | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tel | 0 | 6 | 1 | 0 | 0 | 0 | 0 | 2 | 1 | 0 |
+| tgk | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| tgl | 0 | 3 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| tha | 0 | 8 | 0 | 0 | 0 | 1 | 0 | 3 | 0 | 0 |
+| tir | 0 | 2 | 2 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| tpi | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tsn | 0 | 3 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| tso | 0 | 3 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| tuk | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tum | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tur | 0 | 7 | 0 | 0 | 0 | 2 | 0 | 3 | 0 | 0 |
+| twi | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tyv | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| tzm | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| uig | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| ukr | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| umb | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| urd | 0 | 8 | 1 | 0 | 0 | 0 | 0 | 1 | 1 | 0 |
+| uzn | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| vec | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| ven | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| vie | 1 | 6 | 0 | 0 | 0 | 1 | 0 | 3 | 0 | 0 |
+| war | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| wol | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| xho | 0 | 3 | 2 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ydd | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| yor | 0 | 3 | 2 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| yue | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| zho | 0 | 2 | 0 | 0 | 0 | 1 | 0 | 5 | 0 | 0 |
+| zsm | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| zul | 0 | 3 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
```

Comment:
Do we want a total at the end (or at the top)

Reply:
At the end works.

Reply:
Added at the end.